CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to insta and cargo-insta are documented here.
 
+- Support specifying `cargo-nextest` bin with `INSTA_CARGO_NEXTEST_BIN`.  #721
```

Comment:
+ feel free to add a GH tag or name, I'm trying to encourage this in contributors. No obligation though

Reply:
Sorry, I just mean 
```suggestion
- Support specifying `cargo-nextest` bin with `INSTA_CARGO_NEXTEST_BIN`.  #721 (Louis Fruleux)
```

Reply:
Ohh, my bad tag as in nickname. I was thinking tag as GH tags.

Happily committed